### PR TITLE
matchFilterBtns

### PR DESCRIPTION
**Pull request by SchuylerL**
- Made the match filter buttons work in products.js for all, armor, weapons, potion. Deleted those excess files.
- Left classProducts.js and route as it's own button because it has different code.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -12,11 +12,6 @@ import { CreateUserForm } from './components/CreateUserForm';
 import UserProfile from './components/UserProfile';
 import EditProduct from './components/EditProduct';
 import UserCart from './components/UserCart';
-import Armor from './components/Armor';
-import Weapon from './components/Weapon';
-import Potion from './components/Potion';
-import ClassProducts from './components/ClassProducts';
-import CreateProduct from './components/CreateProduct';
```
Comment (nicknapoli82) [inline src/App.js:16]: Should we leave CreateProduct as an import in App.js? 
Is this the thing that Thom made for the admin feature?

**Review by nicknapoli82 — COMMENTED** [review]
Everything looks good here, I'm just not sure about removing CreateProduct as an import from App.js.
Was this on purpose? Maybe its dead code?

Also good job extending the Products tests. I think everything looks good.

File: src/App.js (hunk 1 above)
Comment (nicknapoli82) [reply src/App.js:16]: I took a look at Thom's most current PR and it looks like he is relying on this import for CreateProduct. I think his merging his branch will resolve this. So I'm going to go ahead and merge this branch into dev.

**Review by nicknapoli82 — DISMISSED** [review]

**Comment by nicknapoli82** [comment]
Oh. Except I just now noticed that both our Travis tests are failing now for some reason.
Need to goto work now, but maybe we look at this first before committing to dev

**Review by roboMoose — CHANGES_REQUESTED** [review]
Nice work on slimming down those files!

Yes - The createProducts component shouldn't be removed. I think this is just a typo.

I am also a little confused about the re-written Products test. It is no longer testing the thing that it was meant to test (i.e. rendering the <Products/> component with some mock data). It looks like you are trying to test a function, but this function is not tied to anything on component and so it isn't really testing anything. 

We can meet and discuss front-end testing if you want, and build out some new tests, but please don't modify the tests that are already written.

**Comment by SchuylerL** [comment]
ok I put the import back, but before I git push. about the test, it was failing passing match.params since I changed the Products.js. So I was putting a placeholder to try and let it pass. So we need to find a way for the test to pass re: match.params or params.

**Review by roboMoose — APPROVED** [review]

**Pull request merged**